TldExtract/TldExtract.cs

```diff
@@ -99,7 +99,10 @@ void AddTldRule (Trie root, string [] labels, bool exception)
 					};
 
 					t.matches [label] = m;
-				}
+				}
+				else if (i == 0) {
+					m.ValidTld = true;
+				}

```

Comment:
The rest of the file was auto-formatted CR/LF changes. 103-105 is the fix though